src/lib/routing-tables.js

```diff
@@ -56,7 +56,7 @@ class RoutingTables {
    * @param {Route|RouteData} _route from ledger B→C
    * @returns {Boolean} whether or not a new route was added
    */
-  addRoute (_route) {
+  addRoute (_route, prefix) {
```

Comment:
renamed to `destinationPrefix`. I think it might be better to keep it separate from the route object, because in almost all cases it would be redundant and cause problems if it were not assigned